<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorDialog</class>
 <widget class="QDialog" name="EditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
   <item>
    <widget class="QStackedWidget" name="editorStackedWidget">
     <widget class="QWidget" name="page0"/>
     <widget class="QWidget" name="page1"/>
     <widget class="QWidget" name="page2"/>
     <widget class="QWidget" name="page3"/>
     <widget class="QWidget" name="page4"/>
     <widget class="QWidget" name="page5"/>
     <widget class="QWidget" name="page6"/>
     <widget class="QWidget" name="page7"/>
     <widget class="QWidget" name="page8"/>
     <widget class="QWidget" name="page9"/>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="upButton">
       <property name="text">
        <string>up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="levelLabel">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downButton">
       <property name="text">
        <string>down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorButton">
       <property name="text">
        <string>color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>color</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>solution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="solutionEdit"/>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
